modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
```

Comment:
Sure. From the top of the module description:

A "replicated" variable is a variable for which there is a copy on each locale.
Referencing a replicated variable (in a stylized way, see below)
accesses its copy on the current locale.